src/constructors/keyframes.js

```diff
@@ -14,6 +14,15 @@ export default function keyframes(
   strings: Styles,
   ...interpolations: Array<Interpolation>
 ): Keyframes {
+  /* Warning if you've used keyframes on React Native */
+  if (
+    process.env.NODE_ENV !== 'production' &&
+    typeof navigator !== 'undefined' &&
+    navigator.product === 'ReactNative'
+  ) {
+    console.warn('Helper `keyframes` cannot be used on React Native');
```

Comment:
I'd change the wording slightly to something like "`keyframes` cannot be used on ReactNative, only on the web. To do animation in ReactNative please use Animated."